src/main/java/org/fisco/bcos/web3j/abi/datatypes/Int.java

```diff
@@ -13,6 +14,22 @@ public Int(BigInteger value) {
         this(MAX_BIT_LENGTH, value);
     }
 
+    /**
+     * check if value between MIN_INT256 ~ MIN_INT256
+     *
+     * @param value
+     * @return
+     */
+    public boolean validInt(BigInteger value) {
+        return value.compareTo(Constant.MIN_INT256) >= 0
+                && value.compareTo(Constant.MAX_INT256) <= 0;
+    }
+
+    @Override
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/app/fisco/web3sdk/pullRequest?prid=4732380)